README.md

```diff
@@ -722,6 +722,7 @@ Locales currently supported:
 * **ja:** Japanese.
 * **pt:** Portuguese.
 * **pirate:** American Pirate.
+* **zh:** Chinese.
```

Comment:
Would you mind putting this _before_ `pirate`? I'd like to give priority to the real locales in the list. :smile: